inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/entity/AuditType.java

```diff
@@ -32,9 +32,9 @@ public enum AuditType {
     DORIS("Doris"),
     KUDU("Kudu"),
     POSTGRES("Postgres"),
-    BINLOG("Binlog"),
-    TUBE("Tube"),
-    MYSQL("MySQL"),
+    BINLOG("MYSQL_BINLOG"),
+    TUBE_MQ("TubeMQ"),
+    MYSQL("MYSQL_SQL"),
```

Comment:
```suggestion
    MYSQL("MYSQL_RAW"),
```

Reply:
he name MYSQL_SQL is set by the manager. Changing the name will cause the manager to fail to issue it.